build.sbt

```diff
@@ -20,7 +20,7 @@ developers := List(
 )
 
 scalaVersion := "3.5.2"
-crossScalaVersions := Seq("2.12.19", "2.12.20", "2.13.14", "2.13.15", "3.3.4", "3.5.2")
+crossScalaVersions := Seq("2.12.19", "2.12.20", "2.13.14", "2.13.15", "2.13.16", "3.3.4", "3.5.2")
```

Comment:
it seems like traditionally only last two versions are supported per each major version, so 2.13.14 can be dropped

Reply:
Ah I see. Happy to do that. :) Just wanted to clarify if there is actually a 2.13.16 out there before opening this, I don't understand how is [sbt-scoverage](https://github.com/scoverage/sbt-scoverage/pull/580) supporting that version if I don't yet see it in the [list of available versions](https://www.scala-lang.org/download/all.html). :/

Reply:
https://contributors.scala-lang.org/t/scala-2-13-16-release-planning/6773/21?u=sethtisue
It's common for the new versions to be published before being fully announced, exactly to let the compiler plugin eco system catch up.

Reply:
Ah thank you! I didn't know that, but it makes all the sense. :)
I will update as suggested and move this out of draft.

Reply:
Done.